docs/nextstrain-internal/pathogen-repo-ci.md

```diff
@@ -0,0 +1,54 @@
+# Pathogen Repo CI
+
+Setting up a CI GitHub Action workflow that automatically runs is extremely
+helpful for catching any changes that break pathogen workflows.
+
+The CI GH Action workflow is _not_ intended to check the validity of the results,
+it just ensures that the pathogen workflow still runs to completion.
+
+## Using `pathogen-repo-ci`
+
+Nextstrain maintains a reusable [pathogen-repo-ci][] GitHub Action workflow that
+can be used to create the CI workflow with minimum set up:
+
+```yaml
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  ci:
+    uses: nextstrain/.github/.github/workflows/pathogen-repo-ci.yaml@master
```

Comment:
Presumably once we tag v1 (when? I'd have thought by now?) we'll suggest that here?

_non-blocking_

Reply:
Tagging the repo is on my TODO list; I was waiting for the "smart" `pathogen-repo-ci` to land, which just happened yesterday (and it looks like you've got at least one further outstanding change).